man/git-reauthor.md

```diff
@@ -0,0 +1,83 @@
+git-reauthor(1) -- Rewrite history to change author's identity
+==============================================================
+
+## SYNOPSIS
+
+`git reauthor [<options>]`
+
+## DESCRIPTION
+
+Rewrite history to change author's identity.
+
+Goes through all existing commits and tags to selectively modify the identities:  
+- modify name and/or email  
+- affect author and/or committer identities  
+- specify the new name/email as parameters or use the ones from the Git config  
+- change all objects or only replace those from a specific identity (defined by its email)
+
```

Comment:
Great advice :) By reviewing this and rewriting the docs, I found that many things were not so clear. I added a new example and rewrote many stuff.
Hopefully it's much better now.
Done in commit 68fb3d5